client/my-sites/plans/current-plan/jetpack-checklist/constants.ts

```diff
@@ -7,10 +7,10 @@ import { SiteSlug } from 'client/types';
 /**
  * Returns the localized duration of a task in given minutes.
  *
- * @param  {Number} minutes Number of minutes.
- * @return {String} Localized duration.
+ * @param  minutes Number of minutes.
+ * @return Localized duration.
  */
-export function getJetpackChecklistTaskDuration( minutes ) {
+export function getJetpackChecklistTaskDuration( minutes: number ): string {
```

Comment:
forget not that type aliases are free in terms of runtime

```ts
type Minutes = number;
type LocalizedDuration = string;

export function getJetpackCheclistTaskDuration( minutes: Minutes ): LocalizedDuration {
	…
}
```

use your own judgment if you find it valuable but I know I sure do even with the extra type code. nothing forces this (we could pass a random number and return a random string) but it does stand in the function's API and docs.

Reply:
Thanks for highlighting. I wish we could see aliases:

![Screen Shot 2019-06-05 at 11 55 26](https://user-images.githubusercontent.com/841763/58947853-1c810e80-8789-11e9-93fc-5ed68e6dd94d.png)

It's a nice tool to express families of functionality that work together, but it also feels like overhead that you need to keep track of as a programmer 😕 

I'll keep it in mind 🙂